setup/triggers/amazon.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: single
+title:  "Amazon AWS Pub/Sub"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker pipelines can be triggered from a queue in [Amazon Simple Queue Service (SQS)] (https://aws.amazon.com/sqs/){:target="\_blank"} that receives new notifications from [Amazon Simple Notification Service (SNS)](https://aws.amazon.com/sns/){:target="\_blank"}.
+
+# Prerequisites
+
+You need an AWS Account and the [AWS Command Line Interface (cli)](https://aws.amazon.com/cli/){:target="\_blank"} installed locally and configured with credentials that allow you to manage your AWS cloud resources.
+
+## Creating an Simple Notification Service (SNS) Topic
+
+* Use the AWS Command Line tool or the AWS Console website to create a SNS Topic that will receive the messages destined for the queue and Spinnaker. You can do this by running the following commands:
+
+```
+ bash4.4$ export AWS_TOPIC_NAME=aws-pubsub-test-topic
+ bash4.4$ aws sns create-topic --name ${AWS_TOPIC_NAME}
+ {
+    "TopicArn": "arn:aws:sns:us-east-1:1234567890:aws-pubsub-test-topic"
+ }
+```
+* Make a note of the TopicArn. You will need this to configure Echo via the echo-local.yml file to tell Spinnaker to subscribe to this topic.
+
+## Creating an Simple Queue Service (SQS) Queue
+
+The next step is to create an SQS Queue that will receive the notification messages.
+
+* Use the AWS Command Line tool or the AWS Console website to create an SQS Queue that Spinnaker will listen to. You can do this by running the following commands:
```

Comment:
Actually, spinnaker will do this for you. You just need to put what you'd like the arn to be.

Reply:
Okay, so I've just gone back through and tested all these steps and confirmed that they work in this order. If you just specify the ARN for the queue in the config file, does it automatically handle the permissions to subscribe you to the topic you provide?

Reply:
Yes it should - as long as you actually have the aws permissions to do the subscribe